src/components/ContactDetails.vue

```diff
@@ -29,7 +29,7 @@
 
 				<!-- fullname -->
 				<template #title>
-					<div v-if="isReadOnly">
```

Comment:
in general, we call the classes something related to the section we are editing. In this case, it can be something like `'class=full-name' `or `'class=name-header'`